app/controllers/groups_controller.rb

```diff
@@ -0,0 +1,40 @@
+class GroupsController < ApplicationController
+  before_action :authenticate_user!, only: %i[new create]
+
+  def index
+    @groups = current_user.groups
```

Comment:
- [ ] Hello @shm04, great effort working on your project, But when I tried to run the project on the browser using the `rails s` command, I got an error saying `NoMethodError in GroupsController#index`. I would kindly advise you to check where the error is coming from and fix it to have a perfect working project.

- Video demonstration 👇 


https://github.com/shm04/budget-app/assets/78634841/fc70182f-52f8-4e1e-b3c4-863bf34798db